### ISSUE TITLE
!@Too many alarms (500) registered from uid

### ISSUE THREAD

**Issue by eugvb** [MaikuB/flutter_local_notifications#307]
Hi, as known, max count of scheduled notifies is 500 (on samsung).
But this count is only incrementing while I cancel pending notifies and schedule new.

For example, if I run this code 5 times
```final list = await notifyPlugin.pendingNotificationRequests();
 for (int i = 0; i < list.length; i++) await notifyPlugin.cancel(list[i].id);
 for (int i = 0; i < 100; i++) {
        await notifyPlugin.schedule(...);
  }
```
I will get exception !@Too many alarms (500) registered from uid.
Also I've tried
```
 await notifyPlugin.cancelAll();
 for (int i = 0; i < 100; i++) {
        await notifyPlugin.schedule(...);
  }
```
same result

**Comment by MaikuB** [comment]
I don't have a Samsung to try to reproduce this right now. If you or anyone in the community is able to do so and provide a solution then I'm happy to look at it

**Comment by MaikuB** [comment]
I believe this issue should be addressed by the PR in #318 now. Will close this, feel free to re-open if this is still an issue
